### PR TITLE
removed links

### PR DESCRIPTION
**Pull request by gonyere**
Sorry about that... honestly not sure how those slipped in :D

### CODE REVIEW

**Comment by cboltz** [comment]
This PR does too much. Please only delete `index.html.1`, but keep `index.hmtl`

**Comment by gonyere** [comment]
Added About section, and added back in missing file...

**Comment by cboltz** [comment]
The re-added index.html looks completely different than the original one - is this intentional?

**Comment by gonyere** [comment]
I will get this right... eventually. Sorry for the disaster.

File: _data/locales/en.yml [hunk 1]
```diff
@@ -131,11 +131,26 @@ technologies:
...
    software:
       trollolo:
         name: trollolo
         url: https://github.com/openSUSE/trollolo
       osc-plugin-collab:
         name: osc-plugin-collab
         url: https://github.com/openSUSE/osc-plugin-collab
+about_title: About openSUSE
+about_description: |
+  openSUSE is a Free Software project created around the openSUSE Linux
```
Comment (hellcp) [inline _data/locales/en.yml:144]: I would just say around open source in general, we have stated our distributions and technologies above already, don't really see much of a reason to repeat that

File: _data/locales/en.yml [hunk 2]
```diff
@@ -131,11 +131,26 @@ technologies:
...
+  distributions and a variety of tools including YaST, the Open Build 
+  Service, openQA,Snapper, KIWI, OSEM and many others. We strive to 
+  promote an open and friendly community where everyone is welcome, 
+  regardless of where they live or work in the world. 
+  
+  openSUSE is well known for it's Linux distributions. Tumbleweed 
+  is a tested, rolling release distribution allowing users to have 
+  an up-to-date system that is tested and well supported. For users 
+  who need long-term support openSUSE maintains Leap, with releases 
+  once a year. MicroOS and Kubic are new self-contained transactional
```
Comment (hellcp) [inline _data/locales/en.yml:154]: Kubic is dead now

File: _data/locales/en.yml [hunk 3]
```diff
@@ -131,11 +131,26 @@ technologies:
...
+  openSUSE is a Free Software project created around the openSUSE Linux 
+  distributions and a variety of tools including YaST, the Open Build 
+  Service, openQA,Snapper, KIWI, OSEM and many others. We strive to 
+  promote an open and friendly community where everyone is welcome, 
+  regardless of where they live or work in the world. 
+  
+  openSUSE is well known for it's Linux distributions. Tumbleweed 
+  is a tested, rolling release distribution allowing users to have 
+  an up-to-date system that is tested and well supported. For users 
+  who need long-term support openSUSE maintains Leap, with releases
```
Comment (hellcp) [inline _data/locales/en.yml:153]: Leap is released every 5ish years with a major release, it is updated with a point version every year, it's a bit confusing I guess

**Pull request merged**